Util_Ui.php

```diff
@@ -242,7 +242,7 @@ static public function nonce_field( $action = -1, $name = '_wpnonce', $referer =
 	 * @return string
 	 */
 	static public function get_notification_box( $message, $id = '' ) {
-		if ( !isset( $_GET['page'] ) || ( isset( $_GET['page'] ) && substr( $_GET['page'], 0, 5 ) != 'w3tc_' ) )
+		if ( !isset( Util_Request::get_string( 'page' ) ) || ( isset( Util_Request::get_string( 'page' ) ) && substr( Util_Request::get_string( 'page' ), 0, 5 ) != 'w3tc_' ) )
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.